webcompat/static/css/src/form.css

```diff
@@ -27,6 +27,21 @@
   margin: 0 0 20px;
 }
 
+.form-button.form-button-multiple {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap-reverse;
+  justify-content: space-between;
+  margin-left: -10px;
+  margin-right: -10px;
+  max-width: none;
+}
+
+.form-button.form-button-multiple button {
```

Comment:
As we want to separate 100% the semantics and the styles, we want to styles for html elements. Could you please change `button` to `.button`?